pkg/driverbuilder/builder/builders.go

```diff
@@ -63,6 +68,8 @@ type MinimumURLsBuilder interface {
 }
 
 func Script(b Builder, c Config, kr kernelrelease.KernelRelease) (string, error) {
+	c.SetGCCVersion(b, kr)
```

Comment:
This sets the gcc version for the current build, given the builder (if it implements `GCCVersionRequestor` interface) or the kernelrelease.